### PR TITLE
perf(event attachment): Drop event attachment index without file

### PR DESCRIPTION
**Pull request by nathan-heskia**
This is a follow up PR for [#18004](https://github.com/getsentry/sentry/pull/18004)

SQL query:
```
--
-- Custom state/database change combination
--

DROP INDEX CONCURRENTLY "sentry_eventattachment_project_id_date_added_750bbee8_idx";
 ;
```

### CODE REVIEW

File: src/sentry/migrations/0061_drop_eventattachment_index_without_file.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    dependencies = [
+        ('sentry', '0060_add_file_eventattachment_index'),
+    ]
+
+    operations = [
+        migrations.SeparateDatabaseAndState(
+            database_operations=[
+                migrations.RunSQL(
+                    """
+                    DROP INDEX CONCURRENTLY "sentry_eventattachment_project_id_date_added_750bbee8_idx";
```
Comment (wedamija) [inline src/sentry/migrations/0061_drop_eventattachment_index_without_file.py:36]: Just want to confirm where we got this index name from? When I try deleting this index the migration generates a statement with `DROP INDEX "sentry_eventattachment_project_id_25d761e1f446d2ff_idx";`.

Weirdly, in production I see `sentry_eventattachment_project_id_25d761e1f446d2ff`. Not sure what the inconsistency is about here...

**Comment by mattrobenolt** [comment]
Did we ever apply the first one manually? I don’t recall that being coordinated.

File: src/sentry/migrations/0061_drop_eventattachment_index_without_file.py (hunk 1 above)
Comment (nathan-heskia) [reply src/sentry/migrations/0061_drop_eventattachment_index_without_file.py:36]: it's the name of the index that i get when running `sentry django sqlmigrate sentry 0001`

Comment (nathan-heskia) [reply src/sentry/migrations/0061_drop_eventattachment_index_without_file.py:36]:
```
--
-- Alter index_together for eventattachment (1 constraint(s))
--
CREATE INDEX "sentry_eventattachment_project_id_date_added_750bbee8_idx" ON "sentry_eventattachment" ("project_id", "date_added");
```

**Comment by BYK** [comment]
Closing as this was never applied and we don't think it will be finished at all.

**Pull request closed without merging**